purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -210,6 +213,7 @@ class Purchases internal constructor(
             fetchAndCacheOfferings(identityManager.currentAppUserID, appInBackground = false)
             log(LogIntent.RC_SUCCESS, OfferingStrings.OFFERINGS_UPDATED_FROM_NETWORK)
         }
+        offlineEntitlementsManager.updateProductEntitlementMappingsCacheIfStale()
```

Comment:
same thinking here

Reply:
It will be performed on SDK configuration and app foreground with a 25 hour throttle. It will also have a 5s jittering.

Reply:
👍